docs/quickstart.md

```diff
@@ -0,0 +1,114 @@
+Up and running in 10 minutes!
+----------------------------
+
+LibreTime is quick and easy to install and get running. Follow this guide to go from zero
+to full internet radio station in 10 minutes!
+
+## Prerequisites
+
+- minimum 1 Ghz Processor, 2 GB RAM, and a wired ethernet connection
+- Ubuntu Server 18.04 LTS on bare-metal or cloud VM
+- git, ntp installed and configured with the correct localization
+
+## Preparing the server
```

Comment:
To be honest, it could be either. Stations that are only broadcasting over the internet could benefit to use a VPS install but everyone would do better with a bare metal machine on-premises.

Reply:
Why I ask is because our target audience determines if this section should be here or in the advanced configuration. Maybe @Robbt can weigh in here

Also I'm not sure that everyone would do better with a bare metal machine - podcasters, distributed stations where there are none or multiple studios, etc, but that is beside the point

Reply:
From my experience, installing from source is most convenient. The station that I'm at broadcasts on both FM and online, so having a physical box in our signal flow is a lot easier than using a cloud VM. For other stations that are just starting, using a cloud VM from DigitalOcean or Linode for $10 or $15 might be a lot cheaper and easier to manage - just needing to fund raise $15 per month to cover broadcasting infrastructure.

We could do an install guide for bare metal and a separate install guide for cloud VMs. Those using cloud VMs won't need to do as many of the configurations.

Reply:
Maybe include the statement that the server needs a static IP and link to something in Advanced Config about how to set that up

Reply:
Also because netplan and ufw are Ubuntu specific by default, so these instructions won't work on other OSes without modification or more initial set up